Package.swift

```diff
@@ -1,5 +1,58 @@
+// swift-tools-version:5.1
+
 import PackageDescription
 
 let package = Package(
-    name: "Alicerce"
+    name: "Alicerce",
+    platforms: [
+        .iOS(.v10)
+    ],
+    products: [
+        .library(
+            name: "Alicerce",
+            targets: [
+                "Analytics",
+                "Core",
+                "DeepLinking",
+                "Extensions",
+                "Logging",
+                "Network",
+                "Observers",
+                "PerformanceMetrics",
+                "Persistence",
+                "Resource",
+                "Stores",
+                "View"]
```

Comment:
the closing `]` should be on the next line with the same indent level as `targets: [` 💅